src/components/menu/menu.tsx

```diff
@@ -271,9 +271,10 @@ class MenuItem extends Component<any, any> {
   render(props: any): VNode<any> {
     const badgeType: string | null =
       props.data.badgeType && !props.isSelected(props.data) ? BadgeType[props.data.badgeType] : BadgeType[props.data.badgeType + 'Active'];
+      const isAdvanced = props.data.value == "Advanced captions settings" ? true : false;
```

Comment:
I feel like `MenuItem` might be use by different components, not only by caption menu, so I would recommend add `role` property in <MenuItem /> (with default value "menuitemradio" for backward compatibility) and pass ` <MenuItem role={"menuitem"}  /> from caption menu in case when it needed.

Reply:
done